app/models/user.rb

```diff
@@ -1,2 +1,28 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  # has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :posts, foreign_key: 'author_id', counter_cache: true
+  has_many :comments, dependent: :destroy
+  has_one :like, dependent: :destroy
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def recent_posts(limit = 2)
+    posts.order(created_at: :desc).limit(limit)
+  end
+
```

Comment:
- 👋🏻 @enocol, just a quick note! 📝 The `recent_posts` method is expected to return the three most recent posts, not two, as per the project requirements. Let's make sure we meet the specs! Thanks! 👍🏻